diff --git a/Source/WebKit/Shared/glib/ProcessExecutablePathGLib.cpp b/Source/WebKit/Shared/glib/ProcessExecutablePathGLib.cpp
index 7f16913581ca..153c1814c493 100644
--- a/Source/WebKit/Shared/glib/ProcessExecutablePathGLib.cpp
+++ b/Source/WebKit/Shared/glib/ProcessExecutablePathGLib.cpp
@@ -45,7 +45,6 @@ static String getExecutablePath()
 
 static String findWebKitProcess(const char* processName)
 {
-#if ENABLE(DEVELOPER_MODE)
     static const char* execDirectory = g_getenv("WEBKIT_EXEC_PATH");
     if (execDirectory) {
         String processPath = FileSystem::pathByAppendingComponent(FileSystem::stringFromFileSystemRepresentation(execDirectory), StringView::fromLatin1(processName));
@@ -53,6 +52,7 @@ static String findWebKitProcess(const char* processName)
             return processPath;
     }
 
+#if ENABLE(DEVELOPER_MODE)
     static String executablePath = getExecutablePath();
     if (!executablePath.isNull()) {
         String processPath = FileSystem::pathByAppendingComponent(executablePath, StringView::fromLatin1(processName));
